.gitignore

```diff
@@ -9,4 +9,4 @@ build/
 
 # Directory created by dartdoc
 doc/api/
-web/firebase/**/*.*
+web/firebase/constants.json
```

Comment:
would it make sense to put the firebase constants file under web/assets? I'm thinking somewhere that's more obviously a non-code place.

You might also consider just naming it `firebase-constants.json` and just directly ignoring that filename, regardless of where it is? (I'm thinking that `constants.json` is a bit too generic a name to ignore)

Reply:
Yes that makes sense. I have renamed & added to `.gitignore` wherever the `firebase-constants.json` is present